### PR TITLE
Md starting point

### PR DESCRIPTION
**Pull request by RandomOfNoWhere**
Moving over .md files from a previous re branding PR.
@jss298aj, and sshiiden, and Bioblaze, thanks for reviewing the content of this PR to remove errors before it was officially a PR.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -174,40 +157,32 @@ applicable.
...
-See [Unit testing](https://docs.godotengine.org/en/latest/contributing/development/core_and_modules/unit_testing.html)
-for information on writing tests in Godot's C++ codebase.
+See [Unit testing](https://docs.blazium.app/en/latest/contributing/development/core_and_modules/unit_testing.html)
+for information on writing tests in Blazium's C++ codebase.
 
-## Contributing to Godot translations
+## Contributing to Blazium translations
 
-You can contribute to Godot translations on [Hosted Weblate](https://hosted.weblate.org/projects/godot-engine/),
+You can contribute to Blazium translations on [Hosted Weblate](https://hosted.weblate.org/projects/blazium-engine/),
```
Comment (sshiiden) [inline CONTRIBUTING.md:165]: Translation contribution is not set up yet

**Review by sshiiden — CHANGES_REQUESTED** [review]
In all the documentation links, the `/en/latest` part should be removed, as the website currently does not support language or version selection.

File: README.md [hunk 2]
```diff
@@ -29,14 +29,21 @@ Godot had been developed by [Juan Linietsky](https://github.com/reduz) and
...
+</p>
 
 ## Getting the engine
 
 ### Binary downloads
 
-Official binaries for the Blazium editor and the export templates can be found
-[on the Blazium website]() when available.
+Official binaries for the Blazium editor and the export templates will be found
+[on the Blazium website](https://redotengine.org/download) once it's set up and on the [GitHub page](https://github.com/blazium-engine/blazium) until then.
```
[the remark this answers is not in the document]
Comment (RandomOfNoWhere) [reply README.md]: Fixed, thank you!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -174,40 +157,32 @@ applicable.
...
+- [Bug tracker](https://github.com/Blazium-Engine/blazium-engine/issues): If there is an
   existing issue about a topic you want to discuss, you can participate directly.
   If not, you can open a new issue. Please mind the guidelines outlined above
   for bug reporting.
-- [Feature proposals](https://github.com/godotengine/godot-proposals/issues):
-  To propose a new feature, we have a dedicated issue tracker for that. Don't
-  hesitate to start by talking about your idea on the Godot Contributors Chat
-  to make sure that it makes sense in Godot's context.
+- Feature Proposals should be made on the [Blazium Discord](https://chat.blazium.app) for now.
+- [Blazium](https://github.com/blazium-engine/) on X (Twitter)
```
[the remark this answers is not in the document]
Comment (RandomOfNoWhere) [reply CONTRIBUTING.md]: Fixed, thank you!

File: README.md [hunk 4]
```diff
@@ -46,7 +53,7 @@ for compilation instructions for every supported platform.
 ## Community and contributing
 
 Blazium is not only an engine but an ever-growing community of users and engine
-developers. The main community channels are listed [on the homepage]().
+developers. Please visit our [Discord server](https://discord.gg/vmJMrGAcb2)!
```
[the remark this answers is not in the document]
Comment (RandomOfNoWhere) [reply README.md]: Fixed, thank you!

File: CONTRIBUTING.md (hunk 1 above)
Comment (RandomOfNoWhere) [reply CONTRIBUTING.md]: Fixed, thank you!

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -174,40 +157,31 @@ applicable.
 Feel free to contribute standalone pull requests to add new tests or improve
 existing tests as well.
 
-See [Unit testing](https://docs.godotengine.org/en/latest/contributing/development/core_and_modules/unit_testing.html)
-for information on writing tests in Godot's C++ codebase.
+See [Unit testing](https://docs.blazium.app/en/latest/contributing/development/core_and_modules/unit_testing.html)
```
Comment (Starkium) [inline CONTRIBUTING.md:160]: I don't think we have an /en/ for languages yet

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -145,10 +128,10 @@ scripting APIs, you **must** update the class reference to document those.
 This is to ensure the documentation coverage doesn't decrease as contributions
 are merged.
 
-[Update documentation XML files](https://docs.godotengine.org/en/latest/contributing/documentation/updating_the_class_reference.html)
+[Update documentation XML files](https://docs.blazium.app/en/latest/contributing/documentation/updating_the_class_reference.html)
 using your compiled binary, then fill in the descriptions.
 Follow the style guide described in the
-[Documentation writing guidelines](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html).
+[Documentation writing guidelines](https://docs.blazium.app/en/latest/contributing/documentation/docs_writing_guidelines.html).
```
Comment (Starkium) [inline CONTRIBUTING.md:134]: don't have /en/ for languages yet

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -145,10 +128,10 @@ scripting APIs, you **must** update the class reference to document those.
 This is to ensure the documentation coverage doesn't decrease as contributions
 are merged.
 
-[Update documentation XML files](https://docs.godotengine.org/en/latest/contributing/documentation/updating_the_class_reference.html)
+[Update documentation XML files](https://docs.blazium.app/en/latest/contributing/documentation/updating_the_class_reference.html)
```
Comment (Starkium) [inline CONTRIBUTING.md:131]: don't have /en/ for languages yet

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -174,40 +157,31 @@ applicable.
...
 
-## Contributing to Godot translations
+## Contributing to Blazium translations
 
-You can contribute to Godot translations on [Hosted Weblate](https://hosted.weblate.org/projects/godot-engine/),
-an open source and web-based translation platform.
+In the future you can contribute to Blazium translations, a link will be provide in the future for this.
 
-Please refer to our [editor and documentation localization guidelines](https://docs.godotengine.org/en/latest/contributing/documentation/editor_and_docs_localization.html)
+Please refer to Blazium's [editor and documentation localization guidelines](https://docs.blazium.app/en/latest/contributing/documentation/editor_and_docs_localization.html)
```
Comment (Starkium) [inline CONTRIBUTING.md:167]: don't have /en/ for languages yet

File: COPYRIGHT.txt [hunk 9]
```diff
@@ -27,32 +27,33 @@
...
 License: Expat
 
 Files: ./icon.png
  ./icon.svg
  ./logo.png
  ./logo.svg
-Comment: Godot Engine logo
-Copyright: 2017, Andrea Calabró
+Comment: Blazium Engine logo
+Copyright: 2024, [LOGE (on X/Twitter)
```
Comment (Starkium) [inline COPYRIGHT.txt:56]: what is LOGE

Comment (RandomOfNoWhere) [reply COPYRIGHT.txt:56]: If I recall correctly this is what someone told me was the user name of the person who made the logo, if incorrect please let me know the information I need to put here :)

Comment (Starkium) [reply COPYRIGHT.txt:56]: ok, didn't know if it was meta data or something, all good.

Comment (RandomOfNoWhere) [reply COPYRIGHT.txt:56]: Honestly, I'm not 100% sure this is correct, as I don't recall the specific conversation at this point I got LOGE from, if there is a way to verify this I would appreciate it.

File: CONTRIBUTING.md (hunk 5 above)
Comment (RandomOfNoWhere) [reply CONTRIBUTING.md]: Fixed, thank you!

File: CONTRIBUTING.md (hunk 8 above)
Comment (RandomOfNoWhere) [reply CONTRIBUTING.md]: Fixed, thank you!

File: CONTRIBUTING.md (hunk 7 above)
Comment (RandomOfNoWhere) [reply CONTRIBUTING.md]: Fixed, thank you!

**Comment by RandomOfNoWhere** [comment]
> In all the documentation links, the `/en/latest` part should be removed, as the website currently does not support language or version selection yet.

Thanks I missed this, should be fixed now.

File: COPYRIGHT.txt (hunk 9 above)
Comment (RandomOfNoWhere) [reply COPYRIGHT.txt]: Verified copyright information with copyright holders, see https://discord.com/channels/1294004245489782896/1294475944177831966/1299382663005732935 for more information

**Review by sshiiden — APPROVED** [review]

**Review by Captain-Wet-Beard — APPROVED** [review]

**Comment by Starkium** [comment]
was the logo creator accreditation thing fixed?

**Comment by RandomOfNoWhere** [comment]
> was the logo creator accreditation thing fixed?

Yes, please see the following lines of code, and this discord conversation (https://discord.com/channels/1294004245489782896/1294475944177831966/1299382663005732935):

COPYRIGHT.txt
55: Comment: Blazium Engine logo
56: Copyright: 2024, KenshiroTM, Nicholas Santos Shiden
57: License: CC0

**Review by WhalesState — APPROVED** [review]

**Review by jss2a98aj — APPROVED** [review]

File: README.md [hunk 10]
```diff
@@ -29,14 +29,21 @@ Godot had been developed by [Juan Linietsky](https://github.com/reduz) and
 [Ariel Manzur](https://github.com/punto-) for several years as an in-house engine, used to publish several work-for-hire
 titles. in October 2024, the project was forked due to community dissatisfaction with project direction and politics being expressed by official Godot accounts.
 
-![Screenshot of a 3D scene in the Godot Engine editor](https://raw.githubusercontent.com/godotengine/godot-design/master/screenshots/editor_tps_demo_1920x1080.jpg)
+Blazium was forked from Godot in [October 2024](https://github.com/blazium-engine/blazium/commit/e86a91030a60be7d37e99c7a6069d75181d5138c),
+intending to improve upon Godot in order to fulfill its potential and contribute to the shared
+codebase of both through a more genuinely community-driven model than Godot.
+
+[Keatram - 2D Pixel Cross-Platform MMORPG by Keros](https://kaetram.com)
```
Comment (Bioblaze) [inline README.md:36]: Remove this we need permission from Keros or Kaetram

File: COPYRIGHT.txt [hunk 11]
```diff
@@ -88,6 +89,7 @@ License: Apache-2.0
 Files: ./scene/animation/easing_equations.h
 Comment: Robert Penner's Easing Functions
 Copyright: 2001, Robert Penner
+ 2024-present, Blazium Engine contributors
```
Comment (Bioblaze) [inline COPYRIGHT.txt:92]: here remove this

**Review by Bioblaze — CHANGES_REQUESTED** [review]

**Comment by RandomOfNoWhere** [comment]
Given the rods from god due to redot merge history

**Pull request closed without merging**